interfaces/automated-testing/scd/design/README.md

```diff
@@ -21,3 +21,9 @@ categories:
 
 * [ASTM strategic coordination](astm-strategic-coordination)
 * [U-space](u-space)
+
+## Coverage
+
+The following references analyze the tests coverage of the uss_qualifier automated testing suite:
```

Comment:
At some point, we may want to rephrase this to be something like "analyze requirements coverage" since test coverage often means "which lines of code were run during testing" rather than "which standard requirements are satisfied by the defined tests", but not worth holding up putting this reference in IMO